cylc/uiserver/tests/test_workflow_retrieval.py

```diff
@@ -14,10 +14,13 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import pytest
+import asyncio
 import sqlite3
 
 from cylc.flow.id import Tokens
-from cylc.uiserver.schema import run_task_query, run_jobs_query
+from cylc.uiserver.schema import run_task_query, run_jobs_query, list_elements
+
+pytest_plugins = ('pytest_asyncio',)
```

Comment:
What does this do?

Reply:
I don't think this should be here. We use `pytest-tornasync` instead of `pytest-asyncio` for the uiserver

```suggestion
```

Reply:
Ah, OK. I didn't see this. I'll change it